PatchViewer.html

```diff
@@ -23,6 +23,11 @@
     td {
         padding: 0;
         margin: 0;
+        /* Default: prevent selection */
+        user-select: none;
+    }.selectable {
```

Comment:
Thanks for your fix, it looks good to me. But could you please refine the indentation and add a blank line here to match the style with the other parts?